src/components/GradBackground.tsx

```diff
@@ -1,14 +1,16 @@
 import { ReactNode } from 'react'
+import './styles/grandBackground.css'
```

Comment:
This is tailwind, the correct way to do this is by re-creating the gradient based on the docs available at: https://tailwindcss.com/docs/gradient-color-stops